ompi/include/mpi.h.in

```diff
@@ -571,6 +571,14 @@ typedef MPI_Win_errhandler_function MPI_Win_errhandler_fn
 #define MPI_MAX_PSET_NAME_LEN    OPAL_MAX_PSET_NAME_LEN /* max pset name len */
 #define MPI_MAX_STRINGTAG_LEN    OPAL_MAX_STRINGTAG_LEN /* max length of string arg to comm from group funcs*/
 
+/*
+ * Constants for C code to access elements in Fortran MPI status array.
+ */
+#define MPI_F_STATUS_SIZE OMPI_FORTRAN_STATUS_SIZE    /* Size of Fortran MPI status array */
+#define MPI_F_SOURCE      0                           /* Index for MPI_SOURCE */       
+#define MPI_F_TAG         1                           /* Index for MPI_TAG */
+#define MPI_F_ERROR       2                           /* Index for MPI_ERROR */
```

Comment:
These values really should come from `ompi/include/mpif-values.pl`, not be hard-coded.

This script is invoked by `autogen.pl`:

https://github.com/open-mpi/ompi/blob/59928bb5af51dec5ee7a9f96466ef6fec991deec/autogen.pl#L1461-L1479

(it looks like there's a little left-over infrastructure from when there was more than one script, but you get the idea)

I'm not sure what the right method is to flow the values from `mpif-values.pl` to `mpi.h.in` -- you don't want to just overwrite `mpi.h.in` because:

1. That would cause the git tree to be dirty
2.  `mpi.h.in` is, itself, a template file, which is written at `configure` time (not `autogen.pl` time)

It will probably take a little bit of thought to figure out a scheme for this -- I don't have the cycles to think this through.  Can you propose something?